Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/NewMeterWizard/ConnectionPage.tsx

```diff
@@ -83,7 +83,10 @@ export default function ConnectionPage(props: IProps) {
 
     React.useEffect(() => {
         // Asset Needs a default that isn't the current asset
-        setSelectedAssetKey(props.AllAssets.find(asset => asset.ID !== props.CurrentAsset.ID).AssetKey);
+        let selectedAsset = props.AllAssets.find(asset => asset.ID !== props.CurrentAsset.ID);
+        if (selectedAsset === undefined)
```

Comment:
I am guessing this is only an issue if there are no Assets in the database.
@gcsantos-gpa does that break a bunch of following steps?

Reply:
```suggestion
        let selectedAsset = props.AllAssets.find(asset => asset.AssetKey !== props.CurrentAsset.AssetKey);
```
It might be better to do this comparison with keys rather than ID's, since assets that aren't in the database all share the same ID of 0. Skipping this step shouldn't break anything.